recipe/meta.yaml

```diff
@@ -15,58 +15,64 @@ source:
     - setupext.patch  # [win]
 
 build:
-  number: 1
+  number: 2
   skip: true  # [py<35]
 
 requirements:
-  build:
-    - pkg-config  # [not win]
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-  host:
-    - python
-    - setuptools
-    - numpy >=1.10
-    - python-dateutil
-    - freetype
-    - msinttypes  # [win]
-    - cycler >=0.10
-    - nose
-    - pyparsing
-    - libpng
-    - zlib  # [win]
-    - pyqt  # [not osx]
-    - tk  # [linux]
-    - icu  # [linux]
   run:
-    - python
-    - setuptools
-    - {{ pin_compatible('numpy') }}
-    - cycler >=0.10
-    - python-dateutil
-    - freetype
-    - pyparsing
-    - libpng
+    - {{ pin_subpackage('matplotlib-core', exact=True) }}
     - pyqt  # [not osx]
-    - tk  # [linux]
-    - icu  # [linux]
-    - tornado
-    - kiwisolver
 
-test:
-  imports:
-    - matplotlib
-    - matplotlib.pyplot
-    - matplotlib._image
-    - matplotlib._path
-    - matplotlib._png
-    - matplotlib._tri
-    - matplotlib.backends._backend_agg
-    - matplotlib.ft2font
-    - matplotlib.ttconv
-    - matplotlib.backends._tkagg  # [not win]
-    - mpl_toolkits
-    - pylab
+outputs:
+  - name: matplotlib-core
```

Comment:
The first `name` above is ignored and we are using `matplotlib-split` just as a convention. That means you need a second output that is named `matplolib` that depends on `matplolib-core` and the optional dependencies.

Pinging @msarahan here who knows more about split packages.

Reply:
Yes, that's correct.  See the numpy recipe for reference: https://github.com/AnacondaRecipes/numpy-feedstock/blob/master/recipe/meta.yaml

Reply:
Hm, but running `conda build` locally definitely produced two packages, and they even worked as expected when I installed them into test environments. I used the [implicit metapackages feature](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#implicit-metapackages). Or do I misunderstand something?

Reply:
If it works, you're fine.  That has been fragile in the past, but hopefully we have it tested well enough now.